lib/python/rose/config.py

```diff
@@ -165,8 +166,7 @@ class ConfigNode(object):
 
     """
 
-    __slots__ = ["STATE_NORMAL", "STATE_USER_IGNORED",
-                 "STATE_SYST_IGNORED", "value", "state", "comments"]
+    __slots__ = ["value", "state", "comments"]
 
     STATE_NORMAL = ""
```

Comment:
It is possible that `ConfigNode.STATE_NORMAL` here might solve the issue without having to remove the three variables from `__slots__`

Reply:
This change is absolutely right, the `__slots__` attribute should be a list of only instance variables (my mistake adding class constants).